File: AVL/AVL-java/src/AVL/Node.java
package DSA;

public class Node 
{
  public Node(Data data)
  {
    this.data=data;
    this.left=null;
    this.right=null;
    this.height=1;
  }

  public Node get_right(){return this.right;}
  public Node get_left(){return this.left;}
  public void set_right(Node node){this.right=node;}
  public void set_left(Node node){this.left=node;}
  public Data get_data(){return this.data;}
  public void set_data(Data data){this.data=data;}
  public void set_height(int n){this.height=n;}
  public int get_height(){return this.height;}

  private Data data;
  private Node right,left;
  private int height;
}

File: Binary-Search-Tree/DART/Node.dart
import 'Data.dart';

class Node 
{
  Node(this._data);

  get data=>this._data;
  set data(Data data){this._data=data;}
  get right=>this._right;
  set right(Node? node){this._right=node;}
  get left=>this._left;
  set left(Node? node){this._left=node;}
  get height=>this._height;
  set height(int n){this._height=n;}

  Node? _right,_left;
  int _height=1;
  Data _data;
}
